pkg/subctl/cmd/export.go

```diff
@@ -60,6 +63,7 @@ func init() {
 
 func addServiceExportFlags(cmd *cobra.Command) {
 	cmd.Flags().StringVarP(&serviceNamespace, "namespace", "n", "", "namespace of the service to be exported")
+	cmd.Flags().StringVarP(&rawWeightsString, "clustersWeight", "cw", "", "weights for each source cluster, supports integr weights only. Formatted as <cluster:weight,cluster:weight> for example cluster1:10,cluster2:90")
```

Comment:
Rather than a comma-separated list in a single param, how about allowing multiple params, eq

`--cluster-weight east:10 --cluster-weight west:20`